### PR TITLE
Feature/msit initializer metadata

### PR DESCRIPTION
**Pull request by jumoni**
Add templates and metadata, please review.

### CODE REVIEW

File: tools/initializer/templates/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    description: name of a gauge type
+    value: LoadGaugeT
+
+  - name: gaugeInstanceName
+    description: name of a gauge instance
+    value: LoadG1
+
+files:
+  - model/gauges.yml
+  - model/swim.acme
```
Comment (IndeedPlusPlus) [inline tools/initializer/templates/metadata.yml]: I did not see this file. Remove?

File: tools/initializer/templates/metadata.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  - model/swim.mtd
+  - model/tsp.yml
+  # model/.acmeproject: templates/model/.acmeproject
```
Comment (IndeedPlusPlus) [inline tools/initializer/templates/metadata.yml]: Please remove dead code.

File: tools/initializer/templates/model/gauges.yml.ftl [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#
+# Gauge Type and Gauge Instance Specifications
+#  - time periods generally in milliseconds
+gauge-types:
+  [=gaugeTypeName]:
+    commands:
+      load :
+    setupParams:
+      targetIP:
```
Comment (IndeedPlusPlus) [inline tools/initializer/templates/model/gauges.yml.ftl]: Is it a good idea to leave these fields empty? Should fill in a variable if possible.

File: tools/initializer/templates/rainbow.properties.ftl [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+## System configuration information
+# These properties may be referred to in various files
+# in the target (e.g., Acme, gauges, effectors, probes)
+# and are replaced by Rainbow with the actual values
+customize.system.target.master = ${rainbow.deployment.location}
+customize.system.target.lb = ${rainbow.deployment.location}
+customize.system.target.lb.httpPort = 1081
+customize.system.target.web0 = ${rainbow.deployment.location}
+customize.system.target.web0.httpPort = 1080
+customize.system.target.web1 = 10.0.0.98
```
Comment (IndeedPlusPlus) [inline tools/initializer/templates/rainbow.properties.ftl]: Are these values fixed? Or should we simplify and put variables here?

File: tools/initializer/templates/stitch/utilities.yml.ftl [hunk 5]
```diff
@@ -0,0 +1,9 @@
+model:
+  name: SwimSys
+  type: Acme
+utilities:
+  [=utilName]:
+    label:
```
Comment (IndeedPlusPlus) [inline tools/initializer/templates/stitch/utilities.yml.ftl]: Should these values be empty? Or should we put variables here?

File: tools/initializer/templates/rainbow-gui.properties.tfl [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+rainbow.delegate.beaconperiod = 10000
+rainbow.deployment.factory.class = org.sa.rainbow.core.ports.guava.GuavaRainbowPortFactory
+
+### Rainbow models
+rainbow.model.number=[=number_of_models]
+
+# Rainbow Acme model
+<#list 1..number_of_models as num>
+	rainbow.model.path_[=num] =
```
Comment (IndeedPlusPlus) [inline tools/initializer/templates/rainbow-gui.properties.tfl]: Where is this num?

File: tools/initializer/templates/rainbow-gui.properties.tfl [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+eBus Relay
+
+rainbow.delegate.beaconperiod = 10000
+rainbow.deployment.factory.class = org.sa.rainbow.core.ports.guava.GuavaRainbowPortFactory
+
+### Rainbow models
+rainbow.model.number=[=number_of_models]
+
+# Rainbow Acme model
+<#list 1..number_of_models as num>
```
Comment (IndeedPlusPlus) [inline tools/initializer/templates/rainbow-gui.properties.tfl]: We are using square bucket syntax, right? There should be no `<#` around.

**Comment by IndeedPlusPlus** [comment]
I've added an integration test. Make sure it can pass before requesting more reviews.

**Comment by lennyzhang** [comment]
Bugs fixed, comments resolved. Good for merge.

**Pull request merged**